src/test/index.ts

```diff
@@ -1,22 +1,22 @@
-//
-// PLEASE DO NOT MODIFY / DELETE UNLESS YOU KNOW WHAT YOU ARE DOING
-//
-// This file is providing the test runner to use when running extension tests.
-// By default the test runner in use is Mocha based.
-//
-// You can provide your own test runner if you want to override it by exporting
-// a function run(testRoot: string, clb: (error:Error) => void) that the extension
-// host can call to run the tests. The test runner is expected to use console.log
-// to report the results back to the caller. When the tests are finished, return
-// a possible error to the callback or null if none.
+// //
+// // PLEASE DO NOT MODIFY / DELETE UNLESS YOU KNOW WHAT YOU ARE DOING
```

Comment:
🧹 Looks like this file can be deleted

Reply:
Sure, I can update the PR over the weekend to use npm. The existing code used a weird mix of yarn and npm and I wasn't sure if this was to support a particular build pipieline or something and didn't want to change it unnecessarily, but npm will work just fine - I'll ping you when it's updated.

Reply:
I was not aware of a mix between Yarn and NPM. That, must have been by mistake, my bad.

Reply:
In that case, my best guess is the (Yeoman?) template you originally used to create the extension was designed to work with yarn but worked just fine with npm as long as nobody tried to run the yarn-specific commands in the package.json scripts section.

It's all on npm now; merge whenever you're ready and I'll create the new PR with the typeChunksFromFile feature support. 👍🏼

Reply:
Yes I used Yeoman when I first created this project to get started. And as this was just a simple utility to help with presentations I never used any proper engineering principles. Just sort of hacked it together.